### PR TITLE
Config files for flexible/{mailgun,mailjet}.

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**
Created a pull request just to test on travis-ci.org.

### CODE REVIEW

**Comment by tmatsuo** [comment]
LGTM

File: appengine/standard/mailgun/test/DeployAppEngineFlexTest.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        chdir($tmpDir);
+        $indexPhp = file_get_contents('index.php');
+        $indexPhp = str_replace(
+            'MAILGUN_DOMAIN_NAME',
+            getenv('MAILGUN_DOMAIN_NAME'),
+            $indexPhp
+        );
+        $indexPhp = str_replace(
+            'MAILGUN_API_KEY',
+            getenv('MAILGUN_APIKEY'),
```
Comment (bshaffer) [inline appengine/standard/mailgun/test/DeployAppEngineFlexTest.php]: We should make these consistent!

File: appengine/standard/mailjet/test/DeployAppEngineFlexTest.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    public function beforeDeploy()
+    {
+        $tmpDir = FileUtil::cloneDirectoryIntoTmp(__DIR__ . '/..');
+        FileUtil::copyDir(__DIR__ . '/../../../flexible/mailjet', $tmpDir);
+        self::$gcloudWrapper->setDir($tmpDir);
+        chdir($tmpDir);
+        $indexPhp = file_get_contents('index.php');
+        $indexPhp = str_replace(
+            'MAILJET_API_KEY',
+            getenv('MAILJET_APIKEY'),
```
Comment (bshaffer) [inline appengine/standard/mailjet/test/DeployAppEngineFlexTest.php]: We should make these consistent!

File: appengine/standard/mailgun/test/DeployAppEngineFlexTest.php (hunk 1 above)
Comment (SurferJeffAtGoogle) [inline appengine/standard/mailgun/test/DeployAppEngineFlexTest.php]: Done.

File: appengine/standard/mailjet/test/DeployAppEngineFlexTest.php (hunk 2 above)
Comment (SurferJeffAtGoogle) [inline appengine/standard/mailjet/test/DeployAppEngineFlexTest.php]: Done.

**Comment by bshaffer** [comment]
LGTM (once tests pass)

**Pull request merged**